### ISSUE TITLE
rendering <del> with strikethrough

### ISSUE THREAD

**Issue by jtonra** [evt-project/evt-builder#40]
Hello EVT,

I'm having trouble rendering `<del>` as a strikethrough. Here is some sample encoding:

```
<div type="stanza">
                <l n="13">Were all unprized uncourted then</l>
                <l n="14">And all the joy one spirit showed</l>
                <l n="15">The other <del>warmly</del><add>deeply</add> felt again</l>
                <l n="16">And friendship like a river flowed</l>
                <l n="17">Constant and strong its silent course</l>
                <l n="18">For naught withstood its gentle force</l>
            </div>
```

I've added some different code to the `custom-style.css` file to try to get this to render, but nothing has worked so far, e.g.

```
.del {
  text-decoration: line-through;
}
.DEL {
  text-decoration: line-through;
}
span.del {
  text-decoration: line-through;
}
```

I would be grateful for any advice!

Justin.

**Comment by RobertoRDT** [comment]
Hi Justin,
in the diplomatic level EVT 2 should rend a deletion using strikethrough by default, with no need for a custom CSS rule. What do you get using &lt;del&gt; without any CSS?

**Comment by jtonra** [comment]
Thanks for your reply, Roberto: I appreciate it! My initial error was that I was working with `"defaultEdition": "critical"` but even after I switched that to `diplomatic`, `<del>` was not rendering with a strikethrough. I returned to the custom styles CSS file and did some tinkering; I got the desired result (image attached) by adding:
```
.ADD {
  vertical-align: super;
  font-size: smaller;
}
.DEL {
  text-decoration: line-through;
}
```
Thanks again for your help; sorry to bother you!
![evt_del_add](https://github.com/evt-project/evt-builder/assets/20701646/63b307d3-85ae-45a2-b776-15705bdfbe74)
